cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -2012,7 +2015,7 @@ func generateMountsModeRun(mst *initramfsMountsState) error {
 	isRunMode := true
 
 	// 2. mount ubuntu-seed (optional for classic)
-	systemdOpts := &systemdMountOptions{
+	seedMountOpts := &systemdMountOptions{
```

Comment:
shouldn't we also mount these nodev,nosuid,noexec, afaik we don't run things from seed? am I wrong?